src/components/MovieDetails/MovieDetailsHeader/MovieDetailsHeader.tsx

```diff
@@ -13,7 +13,7 @@ function MovieDetailsHeader() {
     <div className={styles.movieDetails_headerWrapper}>
       <Logo />
       <button
-        onClick={() => dispatch(handleMovieClick(false))}
+        onClick={() => dispatch(handleMovieClick({} as MovieType))}
```

Comment:
The name does not match the purpose, `seMovieDetails` or smth like that would sound better